projects/packages/forms/src/blocks/form-progress-indicator/save.js

```diff
@@ -0,0 +1,21 @@
+import { useBlockProps } from '@wordpress/block-editor';
+
+const FormProgressIndicatorSave = () => {
+	return (
+		<div
+			{ ...useBlockProps.save( {
+				className: 'jetpack-form-progress-indicator',
+			} ) }
+			data-wp-interactive="jetpack/form"
+			data-wp-style----jp-form-progress-value="state.getStepProgress"
+			data-wp-init="actions.initializeProgress"
```

Comment:
Let's avoid saving these interactivity API bits in HTML and add them at render, see @luisherranz [similar comment here](https://github.com/WordPress/gutenberg/pull/50664#issuecomment-2518456759).

Instead of a `render_block_*` filter noted in comment, you could also use a callback at `register_block_type_from_metadata`.